src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -647,4 +651,13 @@ public ConsoleLogsPageState ConvertViewModelToSerializable()
     {
         return new ConsoleLogsPageState(PageViewModel.SelectedResource?.Name);
     }
+
+    protected override Dictionary<string, AspireTelemetryProperty> GetTelemetryProperties()
+    {
+        return new Dictionary<string, AspireTelemetryProperty>
+        {
+            { TelemetryPropertyKeys.ConsoleLogsApplicationName, new AspireTelemetryProperty(PageViewModel.SelectedResource?.Name ?? string.Empty) },
```

Comment:
Is it ok to record the resource name? That's PII.

Reply:
It's ok to, the property just has to be marked as PII and it'll get hashed and treated like other PII telemetry. Added Pii type, thanks